index.html

```diff
@@ -44,7 +233,47 @@ <h1> I’m George
                 <li> <a href="#"><img src="img/Vectorch.png" alt="icon"></a> </li>
             </ul>
         </div>
+        <div class="action">
+            <div class="normal-btn">
+                <a href="#" class="btn-text">Get my resume</a>
```

Comment:
- [ ] Please give this button the correct styles so that it resembles the one on the Figma design 👇 
Figma 
![image](https://user-images.githubusercontent.com/90222110/192118328-c1c5303c-c132-451e-9891-3cf083289466.png)
**Your design**
![image](https://user-images.githubusercontent.com/90222110/192118336-12d92e34-40e6-42ac-803d-c19b34bbeddf.png)